### ISSUE TITLE
Change dashboard title to Ska Packages Dashboard?

### ISSUE THREAD

**Issue by jeanconn** [sot/skare3_tools#72]
Right now the title is "Ska Packages" and as silly as it sounds, I think this is mostly why I can't get my browser to ever find the page in history.

I'm also not sure where the dashboard is linked except for the skare3 old wiki directions.

**Comment by javierggt** [comment]
Will do.

I don't think it is linked from anywhere.

**Comment by taldcroft** [comment]
I always just type "dashboard" in my browser and it pops right up.

**Comment by jeanconn** [comment]
OK.  Well it doesn't work for me on Linux chrome on the public version of the page unless I type "Packages".  But I confess I've forgotten what content is at 

https://cxc.cfa.harvard.edu/mta/ASPECT/skare3/dashboard/

and what is at 

https://icxc.cfa.harvard.edu/aspect/skare3/dashboard/

**Comment by javierggt** [comment]
My mistake for not linking from somewhere. I just created a link in the aspect wiki.

The page in icxc is an old version, as could be seen from the date (last April). I removed it and replaced it by a redirect. I will still keep both folders, because sometimes I put things in icxc. For example, test logs are always in cxc. Coverage reports of private packages, if there are any, would also go in icxc.

**Comment by taldcroft** [comment]
My chrome hint is to bookmark the page and toss it somewhere like Other Bookmarks. Then chrome seems to be pretty reliable about getting back to the page from the URL. In general for key pages like this I do tend to make and use actual bookmarks.

**Comment by jeanconn** [comment]
Are you suggesting that 'dashboard' shouldn't be in the title of the
dashboard?  At this point it is fine either way; I'm now trained to search
for 'packages'.

**Comment by taldcroft** [comment]
No, dashboard is fine, just suggesting a workaround for the moment.
